example/lib/main.dart

```diff
@@ -96,9 +97,19 @@ class _MyHomePageState extends State<MyHomePage> {
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
-        child: TerminalView(
-          terminal: terminal,
-          style: TerminalStyle(fontFamily: ['Cascadia Mono']),
+        child: Listener(
+          onPointerDown: (e) {
+            if (e.kind == PointerDeviceKind.mouse && e.buttons == kSecondaryMouseButton) {
+              final text = terminal.selectedText;
+              terminal.clearSelection();
+              terminal.refresh();
+              Clipboard.setData(ClipboardData(text: text));
+            }
+          },
+          child: TerminalView(
+            terminal: terminal,
+            style: TerminalStyle(fontFamily: ['Cascadia Mono']),
+          ),
```

Comment:
I think this should be a separate PR as it
- is separated from fixing the selection problem
- may have to be discussed further (not sure if we want to mimic the Windows cmd behavior without any flag or option)

Reply:
I know that